src/main/java/Deadline.java

```diff
@@ -0,0 +1,27 @@
+public class Deadline extends Task{
+    private String time;
+
+    public Deadline(String answer)
```

Comment:
Could you explain the code in your constructor? Because (as far as I know) a constructor should only be used to create an object, assign values to attributes and call other class methods (if needed). I think you could consider abstracting the rest of the code into a separate printTask function.

Reply:
Because some sentences are only printed out when constructed, I put them into the constructor. I will try to seperate them.